src/main/java/org/mule/extension/http/internal/listener/HttpListenerProvider.java

```diff
@@ -181,6 +186,7 @@ public void initialise() throws InitialisationException {
         .setPort(connectionParams.getPort())
         .setTlsContextFactory(tlsContext).setUsePersistentConnections(connectionParams.getUsePersistentConnections())
         .setConnectionIdleTimeout(connectionParams.getConnectionIdleTimeout())
+        .setSchedulerSupplier(() -> schedulerService.ioScheduler(SchedulerConfig.config().withName("http-listener-scheduler-io")))
```

Comment:
add data to the scheduler of the name that can help disambiguate the schedulers when there are more than one listeners (protocol, host and port, for instance)